src/main/java/com/wishlist/services/ProductService.java

```diff
@@ -28,6 +28,7 @@ public Product getProduct(Long id) {
 
     @Transactional
     public Product addProduct(Product product) {
+        product.setStatus(false);
```

Comment:
It's better to use the default parameter in the constructor. Overload the Product constructor and set status as false if you want, but IMO it would be better idea to have status as Enum anyways.